incubator/013-signalstop/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM quay.io/fedora/fedora:35
+COPY demo-signal.sh /demo-signal.sh
+STOPSIGNAL SIGINT
+ENTRYPOINT ["/demo-signal.sh"]
```

Comment:
* https://docs.docker.com/engine/reference/builder/#entrypoint
* https://docs.docker.com/engine/reference/builder/#cmd

Reply:
```suggestion
CMD ["/demo-signal.sh"]
```

Reply:
Thanks! I will exchange the entrypoint by a cmd instruction.